app_legacy/Helpers/database/user.php

```diff
@@ -3,27 +3,16 @@
 use LegacyApp\Community\Enums\ClaimStatus;
 use LegacyApp\Community\Enums\TicketState;
 use LegacyApp\Site\Enums\Permissions;
+use LegacyApp\Site\Models\User;
 
-function GetUserData($user): ?array
+function GetUserData(string $username): ?array
 {
-    sanitize_sql_inputs($user);
-
-    $query = "SELECT * FROM UserAccounts WHERE User='$user'";
-    $dbResult = s_mysql_query($query);
-
-    if (!$dbResult || mysqli_num_rows($dbResult) != 1) {
-        log_sql_fail();
-
-        // failed: Achievement $id doesn't exist!
-        return null;
-    }
-
-    return mysqli_fetch_assoc($dbResult);
+    return User::firstWhere($username)?->toArray();
```

Comment:
This still doesn't work.
```
Unknown column 'Jamiras' in 'where clause'
```
You need `firstWhere('User', $username)`

Reply:
Oh my, thanks! Fixed. I should get more sleep ...